src/app/views/results.html

```diff
@@ -1,10 +1,46 @@
-<div class="consigna__container">
+<!--<div class="container">
   <h2>ACA TIENEN QUE CARGAR UNA GRILLA CON LOS ARTISTAS QUE APARECIERON COMO RESULTADO DE LA BUSQUEDA ANTERIOR</h2>
   <img src="https://i.scdn.co/image/445ffa41207f321883fb01f406165801c7f41ef0" alt="">
   <h4>Cyndi Lauper</h4>
   <img src="https://i.scdn.co/image/6b86adfb6637e08f76390266b617120e95ad16c0" alt="">
   <h4>Cindy Bradley</h4>
   <img src="https://i.scdn.co/image/82633230315fb6b7cc7f4b573bc9e8edf0da69da" alt="">
   <h4>Cindy Bullens</h4>
-  
+</div>-->
+<div>
+    <nav class="logo-center">
+        <img src="https://lh3.googleusercontent.com/UrY7BAZ-XfXGpfkeWg0zCCeo-7ras4DCoRalC_WXXWTK9q5b0Iw7B0YQMsVxZaNB7DM=w300" alt="">
+    </nav>
+    <div class="inside-container">
+        <div class="row">
+            <h1 class="title">Artists</h1>
+            <p>You are currently searching: {{name}} </p>
+        </div>
+        <div class="row">
+            <input type="text" class="search-input" placeholder="Type the name of your favourite artist" ng-model="name">
+            <img src="http://www.redcross.ca/CRC/images/icon-search.png" id="search-icon">
+        </div>
+        <div class="row">
+            <a href="">Home</> <a>Artists</>
+        </div>
+        <div class="row">
+            <div class="itemArtist">
```

Comment:
css classes shouldnt be named with camel case, instead name them like `item-artist`